packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright JS Foundation and other contributors, http://js.foundation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<script type="text/html" data-help-name="inject">
+<p>Injeta uma mensagem em um fluxo manualmente ou em intervalos regulares. A mensagem
+  a carga útil pode ser de vários tipos, incluindo strings, objetos JavaScript ou a hora atual.</p>
+<h3>Saídas</h3>
+<dl class="message-properties">
+    <dt>payload<span class="property-type">vários</span></dt>
+    <dd>A carga configurada da mensagem.</dd>
+    <dt class="optional">topic <span class="property-type">string</span></dt>
+    <dd>Uma propriedade opcional que pode ser configurada no nó.</dd>
+</dl>
+<h3>Detalhes</h3>
+<p>O nó Inject pode iniciar um fluxo com um valor de carga útil específico.
+A carga útil padrão é um carimbo de data / hora da hora atual em milissegundos desde 1º de janeiro de 1970. </p>
```

Comment:
Change to:
`O conteúdo padrão é um estampa de data / hora da momento atual em milisegundos desde 1º de janeiro de 1970. </p>`